tutorials/nservicebus-sagas/2-timeouts/Snippets/Core_7/BuyersRemorseCancelOrderHandling/BuyersRemorseCancelOrderHandling.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace Core_7.BuyersRemorseCancelOrderHandling
+{
+    using NServiceBus;
+    using NServiceBus.Logging;
+    using System;
+    using System.Threading.Tasks;
+
+    #region BuyersRemorseCancelOrderHandling
+
+    class BuyersRemorsePolicy : Saga<BuyersRemorseState>,
```

Comment:
For reference later on, the saga is on the same line. Has my preference, but every other occassion it's the other way around. Again, I prefer this version.

Reply:
Well, inheritance on the next line seems like the standard for messages too etc. I'm not against using a single line but it should be consistent on all types.

What do you think @mauroservienti ?